app/services/recaptcha_annotator.rb

```diff
@@ -5,15 +5,15 @@ class RecaptchaAnnotator
 
   # See: https://cloud.google.com/recaptcha-enterprise/docs/reference/rest/v1/projects.assessments/annotate#reason
   module AnnotationReasons
-    INITIATED_TWO_FACTOR = :INITIATED_TWO_FACTOR
-    PASSED_TWO_FACTOR = :PASSED_TWO_FACTOR
-    FAILED_TWO_FACTOR = :FAILED_TWO_FACTOR
+    INITIATED_TWO_FACTOR = 'INITIATED_TWO_FACTOR'
+    PASSED_TWO_FACTOR = 'PASSED_TWO_FACTOR'
```

Comment:
Is there a reason we want these to by strings instead of symbols?

Reply:
The changes here are necessary to support using this as a value when saving the record, since they map to the [record's enum values](https://github.com/18F/identity-idp/pull/11883/files#diff-5280566e94f53c1fbc27fd71ef599e2aa58e775820dc431961bfd102a497b40fR4-R13). As symbols, this would error when trying to set a value, e.g. `assessment.annotation = :LEGITIMATE`.

It shouldn't have any practical impact since the only other use of these values is in what's included in the API payload to Google, which already had been converting the symbols to strings anyways.